src/schema/v2/artwork/index.ts

```diff
@@ -339,11 +339,19 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
         type: new GraphQLList(InquiryQuestionType),
         description:
           "Structured questions a collector can inquire on about this work",
-        resolve: ({ id }, _params, { inquiryRequestQuestionsLoader }) => {
-          return inquiryRequestQuestionsLoader({
-            inquireable_id: id,
-            inquireable_type: "Artwork",
-          })
+        resolve: (
+          { inquireable, id },
+          _params,
+          { inquiryRequestQuestionsLoader }
+        ) => {
+          // Sale artworks are not inquirable
+          if (!inquireable) {
```

Comment:
shouldn't this be the inverse? i.e. if the artwork is inquireable then we should fetch the inquiry questions?

Reply:
I took a look at this again and tested with an `artwork` and a `saleArtwork` and they were both returning `inquirable` as `true`. It looks like the only way to differentiate between the two is to check for the `sale_ids` array